test/context.test.js

```diff
@@ -0,0 +1,57 @@
+const context = require("../src/context");
+
+test('Get Process Detail Expanded Data With invalid Env vars', () => {
+    process.env.PROCESS_EXECUTION_USER_TOKEN = "sdfklj";
+    process.env.PROCESS_UUID = "lsdkfj";
+    expect(context.getProcessDetailExpandedData()).toStrictEqual(JSON.parse(`{ "Success" : "False", "Status" : "404"}`));
+});
+
+
+// To Do: Add more tests once there are valid endpoints
+
+
+/**
+test('Get Process', () => {
```

Comment:
I'm not sure how Jeff feels about this, but I generally prefer not to have commented-out code in master. In my mind master should be a pristine (or as pristine as possible) working copy of the codebase.

Could you put these tests in another branch and open a separate PR? Then we can merge them once the other endpoints are up.

Reply:
Made a separate pr for endpoint tests once they're up